lib/pc_point.c

```diff
@@ -328,20 +323,27 @@ pc_point_to_geometry_wkb(const PCPOINT *pt, size_t *wkbsize)
 	uint8_t *wkb, *ptr;
 	uint32_t srid = pt->schema->srid;
 	double x, y, z, m;
+	int has_x = pc_point_get_x(pt, &x);
```

Comment:
Pretty minor, but I think it would make more sense to use

```c
int has_x = pc_point_get_x(pt, &x) == PC_SUCCESS;
```

Making it explicit what the return value of `pc_point_get_x` is.